src/api/middleware/requireAuth.ts

```diff
@@ -0,0 +1,9 @@
+import { Handler } from 'express';
+
+export const requireAuth: Handler = (req, res, next) => {
+  if (req.signedCookies?.authed === 'yes') {
```

Comment:
The only thing that concerns me a little about this is that we lose the ability to use Postman... thoughts on using header OR cookies instead of just the cookies?